modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/SlaCard/index.js

```diff
@@ -0,0 +1,197 @@
+					first: slaRawData[0],
+					second: slaRawData[1],
+					third: slaRawData[2],
+				};
+				if (
+					slaRaw.first?.slaLabel === slaPathLabel.current &&
+					(slaRaw.second?.slaTitle || slaRaw.third?.slaTitle) ===
+						slaRaw.first?.slaTitle
+				) {
+					slaFiltedData.push(slaRaw.first);
+
+					if (
+						slaFiltedData[0].slaTitle === slaRaw.second.slaTitle &&
+						slaRaw.second.slaLabel === slaPathLabel.expired
+					) {
+						slaFiltedData[0].slaDateStart =
+							slaRaw.second.slaDateStart;
+					}
+					if (
+						slaFiltedData[0].slaTitle === slaRaw.second.slaTitle &&
+						slaRaw.second.slaLabel === slaPathLabel.future
+					) {
+						slaFiltedData[0].slaDateEnd = slaRaw.second.slaDateEnd;
+					}
+					if (slaRaw.third) {
+						if (
+							slaRaw.third.slaTitle === slaFiltedData[0].slaTitle
+						) {
+							slaFiltedData[0].slaDateEnd =
+								slaRaw.third.slaDateEnd;
+						} else {
+							slaFiltedData.push(slaRaw.third);
+						}
+					}
+				} else {
+					slaRawData.map((sla) => slaFiltedData.push(sla));
+				}
+				setSlaData(slaFiltedData);
+
+				if (!slaSelected) {
+					const slaSelectedCard = slaFiltedData[0].slaLabel;
+					setSlaSelected(slaSelectedCard);
+				}
+			}
+		};
+
+		fetchkoroneikeAccount();
+	}, [accountKey, slaSelected]);
+
+	const handleClick = () => {
+		const slaLength = slaData.length;
+		if (slaSelected === slaData[slaLength - 1].slaLabel) {
+			setSlaSelected(slaData[0].slaLabel);
+		} else if (slaSelected === slaData[0].slaLabel) {
+			setSlaSelected(slaData[1].slaLabel);
+		} else {
+			setSlaSelected(slaData[2].slaLabel);
+		}
+	};
+
+	return (
+		<div className="position-absolute sla-container">
+			<h5 className="mb-4">Support Level</h5>
+
+			{slaData ? (
+				<div>
+					<div
+						className={classNames({
+							'ml-2': slaData[1],
+						})}
+					>
+						<div
+							className={classNames(
+								'align-items-center d-flex sla-card-holder',
+								{
+									'sla-multiple-card ml-2': slaData[1],
+								}
+							)}
+						>
+							{slaData &&
+								slaData.map((sla) => (
+									<SlaCardLayout
+										key={sla.slaTitle}
+										slaDateEnd={sla.slaDateEnd}
+										slaDateStart={sla.slaDateStart}
+										slaLabel={sla.slaLabel}
+										slaSelected={slaSelected}
+										slaTitle={sla.slaTitle}
+									/>
+								))}
+						</div>
+					</div>
+
+					{slaData[1] && (
+						<div
+							className="btn btn-outline-primary d-none hide ml-3 position-relative rounded-circle"
+							onClick={handleClick}
+						>
+							<svg
+								className="bi bi-arrow-right-short"
+								fill="currentColor"
+								height="26"
+								viewBox="0 0 16 16"
+								width="26"
+								xmlns="http://www.w3.org/2000/svg"
+							>
+								<path
+									d="M4 8a.5.5 0 0 1 .5-.5h5.793L8.146 5.354a.5.5 0 1 1 .708-.708l3 3a.5.5 0 0 1 0 .708l-3 3a.5.5 0 0 1-.708-.708L10.293 8.5H4.5A.5.5 0 0 1 4 8z"
+									fillRule="evenodd"
+								/>
+							</svg>
```

Comment:
This should be an asset apart, add this in Documents & Media to a file and import here
```suggestion
							<img src={`${assetsPath}/assets/<name_file>.svg`} />
```

Reply:
I ask to Designer about this, and for now I changing to ClayIcon like we using on CP project.